### PR TITLE
Homework/2/depo

### PR DESCRIPTION
**Pull request by Fox2510**

### CODE REVIEW

File: homework/depo/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<<<<<<< Updated upstream
```
Comment (kovm) [inline homework/depo/index.js]: Remove these unnecessary lines

File: homework/depo/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  } else if (!(a % 3)) {
+    console.log('Fizz')
+  } else {
+    console.log(a)
+  }
+  a++
+}
+
+
+>>>>>>> Stashed changes
```
Comment (kovm) [inline homework/depo/index.js]: And this line too

**Comment by Fox2510** [comment]
Не пойму как исправить конфликт

**Comment by kovm** [comment]
Try to recreate branch
- Close pull request 
- Save your changes somewhere
- Checkout to master
- Update master __git pull origin master__
- Add your changes to index.js
- Delete your remote branch
- Delete your local branch
- Create a branch again
- Commit
- Push
- Create a pull request

**Pull request closed without merging**